config/mlr/h1n1pdm.yaml

```diff
@@ -19,6 +19,8 @@ model:
   pivot: "C.1.9.3"
   hierarchical: true # Keep hierarchical for analysis across regions
   version: "MLR"
+  xi_prior: true
```

Comment:
But here, `xi_prior` is `true` and not a float. I don't understand how this is supposed to work. Shouldn't this need to be a float here?